action.yml

```diff
@@ -3,7 +3,7 @@ description: 'Cache artifacts like dependencies and build outputs to improve wor
 author: 'GitHub'
 inputs:
   path:
-    description: 'A directory to store and save the cache'
+    description: 'Directories to store and save the cache. Supports pattern matching, multipath and single file cache'
```

Comment:
Can we use the description from the README? We're not restricted to only directories